packages/rest-generator/src/commands/generate-rest-client.ts

```diff
@@ -0,0 +1,88 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+import { resolve } from 'path';
+import { createLogger } from '@sap-cloud-sdk/util';
+import execa from 'execa';
+import Command from '@oclif/command';
+import { flags } from '@oclif/parser';
+import cli from 'cli-ux';
+import { generateRest } from '../generator';
+
+const logger = createLogger({
+  level: 'info',
+  messageContext: 'rest-generator-cli'
+});
+
+export interface GeneratorOptions {
+  inputDir: string;
+  outputDir: string;
+  clearOutputDir?: boolean;
+}
+
+export class GenerateRestClient extends Command {
+  static description =
+    'Generates a Rest client from a openApi service file definition. For SAP solutions, you can find these definitions at https://api.sap.com/.';
+
+  static examples = [
+    '$ generate-rest-client generate-rest-client -i directoryWithOpenApiFiles -o outputDirectory',
+    '$ generate-rest-client generate-rest-client --help'
+  ];
+
+  static version = 'ABC';
```

Comment:
left over?

Reply:
oh, yes I wanted to read the value from the package.json